pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -42,6 +42,17 @@ let
   inherit (import ../../build-support/lib/cmake.nix { inherit lib stdenv; }) makeCMakeFlags;
   inherit (import ../../build-support/lib/meson.nix { inherit lib stdenv; }) makeMesonFlags;
 
+  /**
+    This function returns a derivation.
```

Comment:
@roberth Feedback appreciated :heart: 
The only issue here is that the term `package attribute set` is vague and not defined anywhere. (At least i couldn't find it)

I now just made a first approach that we can use to iterate on and move it into the glossary once it exists.

Reply:
Found it: https://nix.dev/manual/nix/latest/glossary#package-attribute-set

Update:

```suggestion
    This function creates a derivation, and returns it in the form of a [package attribute set](https://nix.dev/manual/nix/latest/glossary#package-attribute-set) that refers to the derivation's outputs.
    
    `mkDerivation` takes many argument attributes, most of which affect the derivation environment, but [`meta`](#chap-meta) and [`passthru`](#var-stdenv-passthru) only directly affect package attributes.
    
    The `mkDerivation` argument attributes can be made to refer to one another by passing a function to `mkDerivation`. See [Fixed-point argument of `mkDerivation`](#mkderivation-recursive-attributes).
```



> I now just made a first approach that we can use to iterate on and move it into the glossary once it exists.

I wouldn't mind opening a PR after; just figured this would be easier. (And it is if you think this suggestion is good now)

I'll stop making suggestions after this one :crossed_fingers: